targets/TARGET_NORDIC/TARGET_NRF5_SDK13/trng_api.c

```diff
@@ -42,29 +42,47 @@
 
 void trng_init(trng_t *obj)
 {
+    (void) obj;
+
     (void)nrf_drv_rng_init(NULL);
```

Comment:
Nice hint. Discussed out-of -band with @kl-cruz . We will keep this as is for this PR.